blocks/adp-share-header/adp-share-header.js

```diff
@@ -0,0 +1,41 @@
+import { decorateIcons } from '../../scripts/lib-franklin.js';
+import { createLinkHref, navigateTo } from '../../scripts/scripts.js';
+
+import {
+  selectAllAssets, deselectAllAssets
+} from '../adp-infinite-results-linkshare/adp-infinite-results-linkshare.js';
+
+function createShareInfoHeader(shareInfoHeader) {
+
+  shareInfoHeader.innerHTML = `
+        <div class="adp-share-header-left">
+          <div class="adp-share-header-share-info">
+            <div class="adp-share-subinfo">
+              <div class="adp-share-select-all">
+                <input type="checkbox" id="select-all-checkbox"/>
+                <label for="select-all-checkbox">Select All</label>
+              </div>
+            </div>
+          </div>
+        </div>
+        </div>
+    `;
```

Comment:
HTML is imbalanced. There is an extra closing div.

Reply:
Done